pandas/core/indexing.py

```diff
@@ -155,6 +155,8 @@ def iloc(self) -> _iLocIndexer:
           DataFrame) and that returns valid output for indexing (one of the above).
           This is useful in method chains, when you don't have a reference to the
           calling object, but would like to base your selection on some value.
+        - A tuple of row and column indexes, e.g. ``0, 1`` or ``[1, 2], [2, 5]`` or
```

Comment:
Maybe add that the tuple elements consist of one of the above mentioned ones instead of given examples

Reply:
Although I would add the parentheses to make it explit they are tuples. e,g, ``([1,2], [2,5])``.

Reply:
Acknowledged the comments and updated the statement.

Reply:
Are we suppose to run around 30 some checks for document updates?